web/app/js/components/ResourceList.jsx

```diff
@@ -89,7 +94,7 @@ export default class PodOwnerList extends React.Component {
           <div>
             <PageHeader header={friendlyTitle + "s"} api={this.api} />
             { _.isEmpty(this.state.metrics) ?
-              <CallToAction numDeployments={_.size(this.state.metrics)} /> :
+              this.renderEmptyMessage(friendlyTitle) :
```

Comment:
something to consider for a subsequent review, probably not for this one. consider modifying the `CallToAction` to consistently appear if no resources are meshed...

`servicemesh.jsx` displays a `CallToAction` if no proxies are found:
```js
if (this.proxyCount() === 0) {
  return <CallToAction numDeployments={this.deployCount()} />;
}
```
...but `ResourceList.jsx` only displays the `CallToAction` if `_.isEmpty(this.state.metrics) == true`, which i think won't happen if i have any resources at all, meshed or unmeshed?

taking this a step further, perhaps `CallToAction` could be generalized to all resource types.

screenshots for reference...

<img width="906" alt="screen shot 2018-04-28 at 12 58 40 pm" src="https://user-images.githubusercontent.com/236915/39399485-022a304e-4ae4-11e8-92b1-e6bc33d6ad9a.png">

<img width="913" alt="screen shot 2018-04-28 at 12 58 46 pm" src="https://user-images.githubusercontent.com/236915/39399486-0aa96550-4ae4-11e8-9561-1706cb918875.png">

<img width="297" alt="screen shot 2018-04-28 at 1 03 38 pm" src="https://user-images.githubusercontent.com/236915/39399513-9848fa24-4ae4-11e8-9cf6-1e9256c8bbfa.png">

Reply:
@siggy yeah I'm with you there! I have a pending branch that refactors the service mesh page to include replication controllers, and I think much of your suggestions are implemented there!